e2e-testing/cypress/functions/DataViewFilterFunctions.js

```diff
@@ -9,6 +9,48 @@ class DataViewFilterFunctions {
     cy.get(DataViewLocators.postPreview).children(DataViewLocators.postItem).should('not.be.empty');
   }
 
+  click_add_post_btn() {
```

Comment:
Move these into another function file, outside of FilterFunctions file. Since these are not filter functions.